### ISSUE TITLE
help pls it not working

### ISSUE THREAD

**Issue by flitchka** [worse-666/csgo_external_ahk_hack#16]
cweapon.ahk
error at line 95.
csgo external hack 1_2_2.ahk
error at line 95 in #include file

**Comment by worse-666** [comment]
I didn't find any errors, can you try to download again?

**Comment by iHunter69** [comment]
weird, i dont seem to be getting any of this as well. i wish if i knew AuthotKey i could help the dev make the cheat more advanced

**Comment by iHunter69** [comment]
> I didn't find any errors, can you try to download again?

i agree, also when the new update comes make sure the FOV also makes the hand further, like we see on YouTube where increasing fov makes hands further.

**Comment by iHunter69** [comment]
> I didn't find any errors, can you try to download again?

hello, code needs to be updated bro, Hazedumper got updated

**Comment by flitchka** [comment]
> I didn't find any errors, can you try to download again?
I downloaded again a bunch of times already, but it always gives this error :(
